release/release_workflow.wdl

```diff
@@ -1,34 +1,25 @@
-# Cromwell Release WDL
-# Part 1 of the release process.
-# See also: https://docs.google.com/document/d/1khCqpOYpkCE95pE4a6VfBIxd2zHPWuMILgpisNKCF6c
```

Comment:
👁

Reply:
I put it in  the `meta` section of the workflow, but yeah it's less visible now :/

Reply:
ah ok that's cool, I just noticed the deletion but didn't read to the end where it is now 🙂